app/app.go

```diff
@@ -297,12 +304,12 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 
 	// NOTE: the IBC mock keeper and application module is used only for testing core IBC. Do
 	// note replicate if you do not need to test core IBC or light clients.
-	mockModule := ibcmock.NewAppModule(scopedIBCMockKeeper)
+	// mockModule := ibcmock.NewAppModule(scopedIBCMockKeeper)
```

Comment:
We might need this for integration tests, don't we? May be this is required only for simapp. Not sure

Reply:
Yeah, it is required only for simapp